src/components/layouts/Sidebar.tsx

```diff
@@ -32,18 +48,69 @@ export default class Sidebar extends React.Component<{}> {
                 </div>
               </div>
             </Link>
-            <Link to="/benchmarks" style={{ textDecoration: 'none' }}>
-              <div>
-                <div className="sidebar-inner">
-                  <img
-                    src="assets/icons/bench-icon.svg"
-                    className="sidebar-inner"
-                    alt="Benchmarks"
-                  />
-                  <div className="sidebar-head sidebar-inner">Benchmarks</div>
-                </div>
+
+            <div
+              onClick={() => {
+                this.BenchmarkSubmenu();
+              }}
+            >
+              <div className="sidebar-inner benchmarking">
+                <img
+                  src="assets/icons/bench-icon.svg"
+                  className="sidebar-inner"
+                  alt="Benchmarks"
+                />
+                <div className="sidebar-head sidebar-inner">Benchmarks</div>
               </div>
-            </Link>
+            </div>
+
+            <div className="benchmark-submenu">
+              {this.state.showBenchmarkSubmenu ? (
+                <div>
```

Comment:
Add the `npm run prettier` in the husky precommit hook

Reply:
@ganeshpatro321 in pre-commit hooks, we should not change anything.since if we change anything, then it is not added in `git add .`. We should only prevent the commit in case of linting or the pretty problem. This can be done only by listing the errors of lint during pre-commit and not fixing then automatically. Since, if we fix in pre-commit, then the updated code is not added, rather the previous code gets committed.
Hope my point is clear.

Reply:
Ooohh, i dint know this, are you sure about this?

Reply:
Yes. Very much sure